src/linkedList.js

```diff
@@ -0,0 +1,145 @@
+"use strict"
+import Node from '../src/node'
+export default class LinkedList {
+  constructor(data){
+    this.head = null
+    this.tail = null
+    this.size = 0
+
+  }
+
+  getHeadNode(){
+    if (!this) {
+      return null
+    }
+    return this.head
+  }
+
+  getTailNode(){
+    if (!this) {
+      return null
+    }
+    return this.tail
+  }
+
+  contains(string){
+    let current = this.head
+    if (this.head.data == string) {
+      return true
+    }
+    current = this.head.next
+    if (current.data == string) {
+      return true
+    }
+    if (current.data !== string && current.next == null) {
+      return false
+    }
+  }
```

Comment:
This method/function called `contains` does not meet the spec because it only returns true if it finds the data in one of the first two nodes.

For example, if your test contained,
```
linkedList.insert('apple')
linkedList.insert('banana')
linkedList.insert('cranberry')

expect(linkedList.contains("apple")).to.equal(true)
expect(linkedList.contains("banana")).to.equal(true)
expect(linkedList.contains("cranberry")).to.equal(true)
```
the "apple" and "banana" tests will pass but the "cranberry" test will fail.


I suggest you start over from scratch with this function.  I suggest using a whiteboard or paper or pseudocode to figure out an algorithm which will find out if a piece of data is in the linked list.